src/components/shared/index.tsx

```diff
@@ -78,3 +78,5 @@ export const CustomSectionInner = Styled.CustomSectionInner
 export const CustomPageDescription = Styled.CustomPageDescription
 export const BlackButton = Styled.BlackButton
 export const FlexWrapper = Styled.FlexWrapper
+
+export * as StatusPanel from './StatusPanel'
```

Comment:
Is that used anywhere?

Reply:
It is now, the imports are changed to used barrel file